### PR TITLE
added new options to CategoriesType

### PR DESCRIPTION
**Pull request by Guite**
| Q                 | A
| ----------------- | ---
| Bug fix?          | no
| New feature?      | yes
| BC breaks?        | no
| Deprecations?     | no
| Fixed tickets     | #3410 
| Refs tickets      | -
| License           | MIT
| Changelog updated | yes

### CODE REVIEW

**Comment by craigh** [comment]
I'll look at this tonight.

I think we need to write some tests for the Category FormType so we know we aren't breaking it with updates. I'm nervous about it because it is complicated.

**Comment by Guite** [comment]
I understand. This change is tested though ;-)

**Comment by craigh** [comment]
still working on this - should have something tonight or tomorrow.

File: src/system/CategoriesModule/Form/Type/CategoriesType.php [hunk 1]
```diff
@@ -51,22 +51,55 @@ public function buildForm(FormBuilderInterface $builder, array $options)
...
                 'Symfony\Bridge\Doctrine\Form\Type\EntityType',
                 [
-                    'label_attr' => ['class' => 'hidden'],
+                    'label_attr' => !$options['expanded'] ? ['class' => 'hidden'] : [],
                     'attr' => $options['attr'],
                     'required' => $options['required'],
                     'multiple' => $options['multiple'],
+                    'expanded' => $options['expanded'],
                     'class' => 'ZikulaCategoriesModule:CategoryEntity',
-                    'property' => 'name',
```
Comment (craigh) [inline src/system/CategoriesModule/Form/Type/CategoriesType.php]: what happened to the `property`?

Comment (Guite) [inline src/system/CategoriesModule/Form/Type/CategoriesType.php]: Replaced by `choice_label`, see http://symfony.com/doc/current/reference/forms/types/entity.html#choice-label

**Pull request merged**